arcade/sections.py

```diff
@@ -593,18 +644,41 @@ def dispatch_keyboard_event(self, event, *args, **kwargs) -> Optional[bool]:
         return EVENT_UNHANDLED
 
     def on_mouse_press(self, x: int, y: int, *args, **kwargs) -> Optional[bool]:
+        """
+        Triggers the on_mouse_press event on the appropiate sections or view
+        :param x: the x axis coordinate
+        :param y: the y axis coordinate
+        :param args: any other position arguments that should be deliverd to the dispatched event
+        :param kwargs: any other keyword arguments that should be delivered to the dispatched event
+        :return: EVENT_HANDLED or EVENT_UNHANDLED, or whatever the dispatched method returns
```

Comment:
As far as I remember there should ideally be a blank line between the description and the parameter list to make the contents more reliable to parse for various tools. We had problems with sphinx related to this in the past and some other tools including the parameters as the description.

Reply:
Done!